styles.css

```diff
@@ -0,0 +1,524 @@
+}
+
+#navigation .logo {
+  color: #f55800;
+  margin-left: 1.5rem;
+}
+
+#navigation .logo h3 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.5rem;
+  color: #fff;
+}
+
+#navigation .navlinks {
+  display: none;
+  margin-right: 1.5rem;
+}
+
+#navigation .navlinks .links {
+  display: flex;
+  align-items: center;
+  gap: 3rem;
+}
+
+#navigation .navlinks .links li a {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.938rem;
+  line-height: 1.25rem;
+  color: #fff;
+}
+
+#navigation .navlinks .links li a .fa-envelope {
+  font-size: 1.5rem;
+}
+
+#navigation .navlinks.open {
+  display: flex;
+  justify-content: start;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  background-color: #1c1a19;
+}
+
+#navigation .navlinks.open .links {
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2rem;
+  margin-top: 5rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links :nth-child(4) {
+  display: none;
+}
+
+#navigation .navlinks.open .links li {
+  border-bottom: 1px solid #6f6c6b;
+  padding: 1rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links li a {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+}
+
+#navigation .toggle {
+  height: 2.188rem;
+  width: 2.188rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  transition: all 0.5s ease-in-out;
+  margin-right: 1.5rem;
+  cursor: pointer;
+}
+
+#navigation .toggle .menu {
+  position: relative;
+  height: 0.25rem;
+  width: 1.875rem;
+  border-radius: 5px;
+  background: #fff;
+  transition: all 0.5s ease-in-out;
+}
+
+#navigation .toggle .menu::before,
+#navigation .toggle .menu::after {
+  content: "";
+  position: absolute;
+  height: 0.25rem;
+  width: 1.875rem;
+  border-radius: 5px;
+  background: #fff;
+  transition: all 0.5s ease-in-out;
+}
+
+#navigation .toggle .menu::before {
+  transform: translateY(-10px);
+}
+
+#navigation .toggle .menu::after {
+  transform: translateY(10px);
+}
+
+#navigation .toggle.open .menu {
+  transform: translateX(-35px);
+  background-color: transparent;
+}
+
+#navigation .toggle.open .menu::before {
+  transform: rotate(45deg) translate(24px, -24px);
+}
+
+#navigation .toggle.open .menu::after {
+  transform: rotate(-45deg) translate(24px, 24px);
+}
+
+#portfolio {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 1rem;
+}
+
+#portfolio .title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 2.5rem 0;
+}
+
+#portfolio .title h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+#portfolio .title hr {
+  margin-top: 1rem;
+  background-color: #1c1a19;
+  width: 90%;
+}
```

Comment:
- [ ] Please add more margin between the `title ` and the `hr` of the work section as it is the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) to stick to the  [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=bFO3LY97dTXnhgBF-0)
 as much as possible (e.g., font, colors, images, text, margins) 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Figma design:

<img width="193" alt="Screenshot 2023-03-24 at 10 51 42 AM" src="https://user-images.githubusercontent.com/44798044/227437885-1a73227f-b60b-4c24-a7ca-5cd02f0dd656.png">

Your deisgn:

<img width="368" alt="Screenshot 2023-03-24 at 10 52 07 AM" src="https://user-images.githubusercontent.com/44798044/227437925-e1dfc03a-92ed-4078-82c1-cf69efa809c6.png">